cmd/kube-proxy/app/options/options.go

```diff
@@ -65,6 +66,7 @@ func NewProxyConfig() *ProxyServerConfig {
 func (s *ProxyServerConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(componentconfig.IPVar{Val: &s.BindAddress}, "bind-address", "The IP address for the proxy server to serve on (set to 0.0.0.0 for all interfaces)")
 	fs.StringVar(&s.Master, "master", s.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig)")
+	fs.StringSliceVar(&s.APIServerList, "api-servers", []string{}, "List of Kubernetes API servers for publishing events, and reading pods and services. Comma separated list of hostnames or host:port pairs.")
```

Comment:
why are we adding connection options outside of kubeconfig? I thought we were trying to unify the code path in the kubelet to get rid of the parallel options. If we want the ability to specify multiple hosts for a cluster, kubeconfig needs to be able to specify that